src/include/iscsi-lio-server/UI_dialogs.rb

```diff
@@ -2562,7 +2562,7 @@ def validate
       rescue Cheetah::ExecutionFailed => e
         if e.stderr != nil
           failed_storage += (lun[3] + "\n")
-          table_remove_lun_item(lun[0])
+          table_remove_lun(lun[3])
```

Comment:
well, this `lun[X]` is a bit unreadable, but I think in this phase we should not change it. But for future it would be nicer to have some name or hash with symbols, so there is at least some clue what it is, not just some anonymous entry in array.

Reply:
Thanks！！！